src/config/cluster-config.c

```diff
@@ -87,18 +165,67 @@ int Read_Cluster(XML_NODE node, void *d1, __attribute__((unused)) void *d2) {
                 Config->hide_cluster_info = 1;
             } else if (strcmp(node[i]->content, "no") == 0) {
                 Config->hide_cluster_info = 0;
-            } else {
+            } else if (output == NULL) {
                 merror(XML_VALUEERR, node[i]->element, node[i]->content);
                 return OS_INVALID;
+            } else {
+                snprintf(message, OS_FLSIZE,
+                    "Invalid value for element '%s': %s.",
+                    node[i]->element, node[i]->content);
+                wm_strcat(output, message, '\n');
+                return OS_INVALID;
             }
         } else if (!strcmp(node[i]->element, interval)) {
-            mwarn("Detected a deprecated configuration for cluster. Interval option is not longer available.");
+            if (output == NULL){
+                mwarn("Detected a deprecated configuration for cluster. Interval option is not longer available.");
+            } else {
+                wm_strcat(output, "WARNING: Detected a deprecated configuration for cluster. Interval option is not longer available.", '\n');
+            }
         } else if (!strcmp(node[i]->element, nodes)) {
+            if (output) {
+                
+                /* Get children */
+                xml_node **children = NULL;
+                    if (children = OS_GetElementsbyNode(xml, node[i]), !children) {
+                return OS_INVALID;
```

Comment:
Before a `return OS_INVALID`, should be an error message. Please, notify the error.